src/Sulu/Bundle/PreviewBundle/Resources/js/views/Preview/stores/PreviewConfigStore.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+import {action, computed, observable} from 'mobx';
+import queryString from 'query-string';
+
+export type PreviewRouteName = 'start' | 'render' | 'update' | 'update-context' | 'stop';
+export type PreviewMode = 'auto' | 'on_request' | 'off';
+
+type PreviewConfigStoreConfig = {
+    routes: { [PreviewRouteName]: string },
+    debounceDelay: number,
+    mode: PreviewMode,
+};
+
+class PreviewConfigStore {
```

Comment:
Maybe we should make this the `PreviewStore`, and let it also make all the requests, which are currently done in the component?